hepcrawl/inputs.py

```diff
@@ -75,3 +75,10 @@ def clean_tags_from_affiliations(value):
 def clean_collaborations(value):
     """Remove the prefixes for collaborations"""
     return value.replace("for the", "").strip()
+
+
+def clean_abstract(text):
```

Comment:
This can be renamed to something more generic as perhaps not only abstracts needs this kind of cleaning. Suggestion: `clean_whitespace_characters`